scalyr_agent/builtin_monitors/docker_monitor.py

```diff
@@ -1234,6 +1245,120 @@ class DockerMonitor( ScalyrMonitor ):
     It can also collect all log messages written to stdout and stderr by those containers, in conjunction with syslog.
     See the online documentation for more details.
 
+    ## Docker Labels
+
+    You can get the Scalyr agent to upload custom attributes for your containers by setting labels on the container, and then configuring the agent to upload labels as attributes.  This is controlled by a number of config options:
+
+    * **labels\_as\_attributes** - When `true` upload labels that pass the include/exclude filters (see below) as log attributes for the logs generated by the container.  Defaults to `false`
+    *  **label\_include\_globs** - A list of [glob strings](https://docs.python.org/2/library/fnmatch.html) used to include labels to be uploaded as log attributes.  Any label that matches any glob in this list will be included as an attribute, as long as it not excluded by `label_exclude_globs`.  Defaults to `[ '*' ]` (everything)
+    *  **label\_exclude\_globs** - A list of glob strings used to exclude labels from being uploaded as log attributes.  Any label that matches any glob on this list will be excluded as an attribute.  Exclusion rules are applied *after* inclusion rules.  Defaults to `[ 'com.scalyr.config.*' ]`
+    *  **label_prefix** - A string to add to the beginning of any label key before uploading it as a log attribute.  e.g. if the value for `label_prefix` is `docker_` then the container labels `app` and `tier` will be uploaded to Scalyr with attribute keys `docker_app` and `docker_tier`.  Defaults to ''
+
+    You can change these config options by editing the `agent.d/docker.json` file. Please [follow the instructions here](https://www.scalyr.com/help/install-agent-docker#modify-config) to export the configuration of your running Scalyr Agent.
+
+    A sample configuration that uploaded the attributes `tier`, `app` and `region`, with the prefix `dl_` would look like this:
+
+    ```
+    monitors: [
+      {
+        "module": "scalyr_agent.builtin_monitors.docker_monitor",
+        ...
+        labels_as_attributes: true,
+        label_include_globs: ['tier', 'app', 'region' ],
+        label_prefix: 'dl_'
+      }
+    ]
+    ```
+
+    Note: Log attributes contribute towards your total log volume, so it is wise to limit the labels to a small set of approved values, to avoid paying for attributes that you don't need.
+
+    ## Using Docker Labels for Configuration
+
+    You can also use docker labels to configure the log settings for a specific container, such as setting the parser or setting redaction rules.
+
+    The agent takes any label on a container that begins with `com.scalyr.config.log.` and maps it to the corresponding option in the `log_config` stanza for that container's logs (minus the prefix). 
+
+    For example, if you add the following label to your container:
```

Comment:
Do they have to turn anything on in the agent to use the configuration-options-from-labels feature?